website/src/content/main-concepts/proxy-to-external-api.mdx

```diff
@@ -16,74 +18,213 @@ Merchant Center applications can make [authenticated requests](/main-concepts/au
+    }),
+  });
+};
+```
+
+## Usage for SDK actions
+
+By default, all requests with the SDK actions are configured to be sent to the Merchant Center API Gateway.
+When making requests to the external API using the SDK actions, you can now use the `forwardTo` object, which wraps the normal action creators and configures them with the required HTTP headers.
+
+```js
+actions.forwardTo.get({ uri: 'https://my-custom-app.com/graphql' });
+actions.forwardTo.del({ uri: 'https://my-custom-app.com/graphql' });
+actions.forwardTo.head({ uri: 'https://my-custom-app.com/graphql' });
+actions.forwardTo.post({
+  uri: 'https://my-custom-app.com/graphql',
+  payload: { say: 'Hello' },
 });
 ```
 
-The request will be then validated from the Merchant Center API Gateway and proxied to the specified HTTP header, sending alongside an `Authorization: Bearer <token>` HTTP header, in the form of a JWT.
+<Info>
+
+As a reminder, action creators can be be dispatched using the `useAsyncDispatch` React hook. You can [check out an example](https://github.com/commercetools/merchant-center-application-kit/blob/e9548422ca8580a5846f80a250ccac043806ee3c/playground/src/components/echo-server/echo-server.js#L28-L52) in the Playground application.
+
+</Info>
+
+# Authenticating requests from the external API
+
+When a valid request is sent via the `/proxy/forward-to` endpoint, the Merchant Center API Gateway forwards the request to the external API with an `Authorization: Bearer <token>` HTTP header.
+
+The bearer token is a short-living JSON Web Token (JWT) that is used exclusively for the authorization exchange between the Merchant Center API Gateway and the external API. The token is valid for **60 seconds**.
+
+The external API must verify that the token is valid by [using the JSON Web Key Set endpoint](#using-json-web-key-set-endpoint).
 
 <Warning>
 
-For security reasons, the exchanged JWT has an expiration date of 60 seconds, as the token should only be used to verify that the request is authorized.
+It's imperative that the external API is securely protected and that only users with access to the project are able to connect to the API.
 
 </Warning>
 
-## Using JSON Web Key Set endpoint
 
-At this point, your backend server will receive the request with the `Authorization` header, which you can use to validate that the JWT is valid.
-The validation is performed by using the `/.well-known/jwks.json` endpoint exposed by the Merchant Center API Gateway. You can read more about JWKS [here](https://auth0.com/docs/jwks) and [here](https://tools.ietf.org/html/rfc7517).
+## Validating the JSON Web Token
 
-The endpoint exposes a public key which can be used to verify the JWT signature, according to the OIDC spec. We also expose a `/.well-known/openid-configuration` discovery endpoint.
+To facilitate the implementation of validating the JWT from the `Authorization: Bearer <token>` HTTP header, we provide a package with built-in functions and helpers to perform the heavy work.
```

Comment:
```suggestion
To facilitate the implementation of validating the JWT from the `Authorization: Bearer <token>` HTTP header, we provide a package with built-in functions and helpers to facilitate the heavy lifting.
```

Reply:
We're repeating `facilitate`. We should use another word

Reply:
True, happy to leave it as is then.